lib/exchange/authorizationCode.js

```diff
@@ -66,37 +67,37 @@ module.exports = function(options, issue) {
     options = undefined;
   }
   options = options || {};
-  
-  if (!issue) { throw new TypeError('oauth2orize.authorizationCode exchange requires an issue callback'); }
-  
+
+  if (!issue) { throw new TypeError(g.f('{{oauth2orize.authorizationCode}} exchange requires an issue callback')); }
+
   var userProperty = options.userProperty || 'user';
 
   return function authorization_code(req, res, next) {
-    if (!req.body) { return next(new Error('OAuth2orize requires body parsing. Did you forget app.use(express.bodyParser())?')); }
-    
+    if (!req.body) { return next(new Error(g.f('{{OAuth2orize}} requires body parsing. Did you forget app.use(express.bodyParser())?'))); }
```

Comment:
I think you should have curly braces for `app.use(express.bodyParser())`
Like `{{app.use(express.bodyParser())}}`